internal/backends/postgresql/metadata/registry.go

```diff
@@ -465,11 +472,18 @@ func (r *Registry) collectionCreate(ctx context.Context, p *pgxpool.Pool, dbName
 	list := maps.Values(colls)
 
 	for {
-		tableName = fmt.Sprintf("%s_%08x", strings.ToLower(collectionName), s)
+		tableName = strings.ToLower(specialCharacters.ReplaceAllString(collectionName, "_"))
```

Comment:
We should call `ToLower` first. Otherwise, "MY_COLLECTION_NAME" will be replaced by "__________________"

Reply:
That is very true, thanks :pray: made the change